<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1248</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>학생 관리</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>871</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_hostname">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Host Name</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_class">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>과목명 :</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>871</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="resultLayout"/>
   </widget>
   <widget class="QPushButton" name="btn_update">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>610</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_tStudents">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_searchByName">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>50</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>이름으로 검색</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_search">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>50</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>검색</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_resetInput">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>50</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>초기화</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>60</y>
      <width>331</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>강제 입력 / 삭제</string>
    </property>
    <widget class="QLabel" name="lbl_forceID">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>학번:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_id">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_name">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_forceName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>이름:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_dept">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_forceDept">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>학과:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_forceInsert">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>강제 입력</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_forceDelete">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>강제 삭제</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>290</y>
      <width>331</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>학점 관리</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>성적 산출:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_pA">
     <property name="geometry">
      <rect>
       <x>38</x>
       <y>70</y>
       <width>27</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_gradeM_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>출석</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_gradeM_3">
     <property name="geometry">
      <rect>
       <x>68</x>
       <y>70</y>
       <width>15</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_gradeM_4">
     <property name="geometry">
      <rect>
       <x>146</x>
       <y>70</y>
       <width>15</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_pH">
     <property name="geometry">
      <rect>
       <x>116</x>
       <y>70</y>
       <width>27</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_gradeM_5">
     <property name="geometry">
      <rect>
       <x>88</x>
       <y>70</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>과제</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_gradeM_6">
     <property name="geometry">
      <rect>
       <x>226</x>
       <y>70</y>
       <width>15</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_pM">
     <property name="geometry">
      <rect>
       <x>196</x>
       <y>70</y>
       <width>27</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>40</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_gradeM_7">
     <property name="geometry">
      <rect>
       <x>166</x>
       <y>70</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>중간</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_pF">
     <property name="geometry">
      <rect>
       <x>276</x>
       <y>70</y>
       <width>27</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>40</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_gradeM_8">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>기말</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_gradeM_9">
     <property name="geometry">
      <rect>
       <x>306</x>
       <y>70</y>
       <width>15</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_calcScore">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>85</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>산출</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_gradeM_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>A+</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_gradeM_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>B+</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_gradeM_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>C+</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_gradeM_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>D+</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_gradeM_14">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>210</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>D0</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_gradeM_15">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>A0</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_gradeM_16">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>B0</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_gradeM_17">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>C0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_dpMin">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>27</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_apMin">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>27</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>90</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_cpMin">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>27</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>30</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_bpMin">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>27</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>60</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_d0Min">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>210</y>
       <width>27</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_b0Min">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>27</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>40</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_a0Min">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>27</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>80</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_c0Min">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>180</y>
       <width>27</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>20</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_dp">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>210</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_ap">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_bp">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_cp">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_d0">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>210</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_b0">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_c0">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>180</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_a0">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_saveExcel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>270</y>
       <width>85</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>엑셀로 저장</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_resetGrade">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>270</y>
       <width>85</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>초기화</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1248</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
